support/elasticsearch_search.erl

```diff
@@ -634,12 +639,26 @@ map_filter([[Key | _] | _] = Filters, Context) when is_list(Key); is_binary(Key)
     }}};
 map_filter([Key, Value], Context) when is_list(Key) ->
     map_filter([list_to_binary(Key), Value], Context);
+
 map_filter([<<"pivot_", _/binary>> = Pivot, Value], Context) ->
     map_filter([map_pivot(Pivot), Value], Context);
+
+%% location_lat and location_lng are both mapped to a field 
+%% called geolocation within elasticsearch
+map_filter([<<"location_lat">>, Value], Context) ->
+    map_filter([<<"geolocation">>,  Value], Context);
+map_filter([<<"location_lng">>, Value], Context) ->
+    map_filter([<<"geolocation">>, Value], Context);
```

Comment:
Edge case: can there be a `geolocation` without either lat or long? In that case we could check for lat and long specifically.

Reply:
I just checked with Loetie, geolocation is of type 'geo_point' and can only be created if both lat and long are supplied, so no!